src/CSI.IBTA.BenefitsService/Controllers/EnrollmentsController.cs

```diff
@@ -0,0 +1,62 @@
+﻿using CSI.IBTA.BenefitsService.Interfaces;
+using CSI.IBTA.Shared.Constants;
+using CSI.IBTA.Shared.DTOs;
+using CSI.IBTA.Shared.Entities;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace CSI.IBTA.BenefitsService.Controllers
+{
+    [Route("api/v1/[controller]")]
+    [ApiController]
+    public class EnrollmentsController : Controller
+    {
+        private readonly IEnrollmentsService _enrollmentsService;
+
+        public EnrollmentsController(IEnrollmentsService enrollmentsService)
+        {
+            _enrollmentsService = enrollmentsService;
+        }
+
+        [HttpPost("{employeeId}")]
+        [Authorize(Roles = $"{nameof(Role.EmployerAdmin)}")]
+        public async Task<IActionResult> GetEnrollmentsByEmployeeId(int employeeId, GetEnrollmentsDto dto)
+        {
+            var employerIdClaim = User.Claims.FirstOrDefault(c => c.Type == JwtTokenClaimConstants.EmployerId);
+
+            if (employerIdClaim == null || !int.TryParse(employerIdClaim.Value, out int employerId))
```

Comment:
the logic of receiving employer ID claim is repeatable and could be extracted to a extension method.